tools/c7n_org/c7n_org/cli.py

```diff
@@ -701,6 +723,8 @@ def run(config, use, output_dir, accounts, not_accounts, tags, region,
         if not os.path.exists(cache_path):
             os.makedirs(cache_path)
 
+    output_dir = initialize_provider_output(custodian_config, output_dir, region)
```

Comment:
Ahhh I think this clears the wall I kept running into (a way to identify bucket region once in a way all the workers would see). Nice, handling it with a query parameter seems friendlier than caching gymnastics or adding special config items :+1: